test/integration/multikueue/multikueue_test.go

```diff
@@ -1050,8 +1050,62 @@ var _ = ginkgo.Describe("Multikueue", ginkgo.Ordered, ginkgo.ContinueOnFailure,
 	})
 
 	ginkgo.It("Should run a MPIJob on worker if admitted", func() {
+		admission := utiltesting.MakeAdmission(managerCq.Name).PodSets(
+			kueue.PodSetAssignment{
+				Name: "launcher",
+			}, kueue.PodSetAssignment{
+				Name: "worker",
+			},
+		)
+		mpijobNoManagedBy := testingmpijob.MakeMPIJob("mpijob2", managerNs.Name).
```

Comment:
Can we move it to separate `It`?

Reply:
we could I guess, I wanted to reduce testing time, keeping it here

Reply:
done